go-apps/meep-virt-engine/server/chart-template.go

```diff
@@ -149,6 +154,31 @@ func Deploy(sandboxName string, procName string, model *mod.Model) error {
 	return nil
 }
 
+func getMepService(proc *dataModel.Process) string {
+	// !!! Temporary patch for MEP Service configuration !!!
```

Comment:
Is it still a temporary patch?

Reply:
It is the selected implementation for now that could/should eventually change. However, I agree that the comment is misleading here. I will remove it.